test/test_zoom.py

```diff
@@ -4,14 +4,19 @@
 import requests_mock
 from parsons import Table, Zoom
 
-API_KEY = "fake_api_key"
-API_SECRET = "fake_api_secret"
+ACCOUNT_ID = "fakeAccountID"
+CLIENT_ID = "fakeClientID"
+CLIENT_SECRET = "fakeClientSecret"
+
 ZOOM_URI = "https://api.zoom.us/v2/"
+ZOOM_AUTH_CALLBACK = "https://zoom.us/oauth/token"
 
 
 class TestZoom(unittest.TestCase):
```

Comment:
Let's add some unit tests for the new functions we added. Should be able to just copy the existing patterns for the new functions